### PR TITLE
Improvements in doctor.js

### LINKED ISSUE #55: Adding more information in doctor.js
## Describe the bug/feature
- There are some information which is important to know for patient about the doctor.

## Parameter added:
-1. Language : Preferred language for effective patient-doctor communication.
-2. Quality Assurance : Involvement in patient safety programs to demonstrate doctor commitment to providing high-quality care.
-3. Email Address: To connect the Doctor and Patient.

I want to work on this under GSSOC'23

### PR DESCRIPTION
**Pull request by Naman1729**
This more modifies the file:
- server/model/doctor.js

This PR resolves #55 

- [x] Adding Language: Preferred language for effective patient-doctor communication.
- [x] Adding qualityAssurance: To providing high-quality care.
- [x] Adding emailAddress: To connect Doctor and Patient.
- [x] Adding clinicHygiene: Rating of hygiene of clinic.
- [x] Adding openTime: Opening time of clinic.
- [x] Adding closeTime: Closing time of clinic.

### CODE REVIEW

File: server/model/doctor.js [hunk 1]
```diff
@@ -10,13 +11,68 @@ const doctorSchema = new mongoose.Schema({
         type: String,
         required: true
     },
+    /**Preferred language for effective patient-doctor communication and enhanced comprehension. */
+    Language: {
```
Comment (apeksha235) [inline server/model/doctor.js:15]: - avoid using capital letters to start a variable, just change this. rest all seems great

File: server/model/doctor.js [hunk 2]
```diff
@@ -10,13 +11,68 @@ const doctorSchema = new mongoose.Schema({
...
+    clinicHygiene: {
+        type: Number, 
+        required:true,
+        min: 1,
+        max: 5
+    },
     education:{
         type:String
     },
     placeOfPractice:{
```
Comment (apeksha235) [inline server/model/doctor.js:72]: - set rest of the parameters as required as well.

**Review by apeksha235 — CHANGES_REQUESTED** [review]
I have asked you to make a few changes, do that and let me know.

**Comment by jain-rishabh-21** [comment]
I suggest using "prettier" vscode extension for formatting! 
Just install prettier and then auto format the file.
@Naman1729

**Review by apeksha235 — APPROVED** [review]
It's good to go!

**Comment by Naman1729** [comment]
Hey @apeksha235, Please add labels of GSSOC'23.

**Pull request merged**